.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install system requirements
+        run: sudo apt install -y git libffi-dev libmysqlclient-dev libssl-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev libxmlsec1-dev liblzma-dev software-properties-common tk-dev xz-utils zlib1g-dev
+      - name: Install pip
+        run: python -m pip install -r requirements/pip.txt
+      - name: Install python dependencies
+        run: pip install -r requirements/edx/development.txt
```

Comment:
can we introduce caching for these dependencies? since this installation is taking most of the time for this check